### ISSUE TITLE
Concentrating icon repeating multiple times and can not be removed.

### ISSUE THREAD

**Issue by emattsen** [death-save/combat-utility-belt#319]
I see a bug in CUB with D&D 5e and FVTT 7.5.  Last night a player cast six concentration spells, and the concentration icon appeared and remained each time.  The icons can not be removed.  If I go to the status effects for the token, I can add and remove a seventh concentrating icon, but I can not remove the first six.  I peeked into the actors.db, and there are six entries for CUB concentrating on this actor. This was to a linked actor.

![image](https://user-images.githubusercontent.com/57371454/97358429-a09dcf80-1871-11eb-8b49-b0b66471dbe1.png)

**Comment by eclarke12** [comment]
@emattsen can you confirm the version of CUB you are using?

As a workaround the following macro should remove the condition:
`game.cub.removeAllConditions()`

**Comment by emattsen** [comment]
CUB version 1.3.2

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by eclarke12** [comment]
Can you please send me an export of your Condition Lab and the affected actor please?

Also, after exporting those items, can you confirm whether you've been into the Condition Lab and saved it? And if not, can you try doing so and let me know if the issue is resolved? Thank you for your time.

**Comment by emattsen** [comment]
Here is the data you requested.  I had already used the macro to clear the conditions from the actor.

I have been into condition lab to look at it, but made no changes, so I did not save it. After going in and making no changes and saving it, it seems to not be duplicating, but I am not getting the chat messages about multiple concentrations spells that I thought I would get.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by emattsen** [comment]
Sorry, the chat notifications did come through, but they seemed delayed.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

From: Erik Mattsen<mailto:emattsen@hotmail.com>
Sent: Wednesday, October 28, 2020 9:47 AM
To: death-save/combat-utility-belt<mailto:reply@reply.github.com>
Subject: RE: [death-save/combat-utility-belt] Concentrating icon repeating multiple times and can not be removed. (#319)

Here is the data you requested.  I had already used the macro to clear the conditions from the actor.

I have been into condition lab to look at it, but made no changes, so I did not save it. After going in and making no changes and saving it, it seems to not be duplicating, but I am not getting the chat messages about multiple concentrations spells that I thought I would get.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10
